DOCS-10220-create-and-implement-nav-partial (#75)
mulesoft/docs-visualizer: modules/ROOT/pages/customize-your-canvas.adoc

endif::[]

> You can choose what appears in your Anypoint Visualizer canvas by applying filters for environments, tags, and services.

Reviewer: ```suggestion
You can choose objects? in your Anypoint Visualizer canvas by applying filters for environments, tags, and services.
```

Author: Not choosing objects - just filtering using the parameters noted at the end.

This enables you to hide the nodes that you do not want to see in the application graph.
Filters have a cascading design, so all nodes that don't belong to the checked environments or tags are automatically hidden from the list of services.

You can toggle *Show dependents and dependencies* to see the services and dependencies of the nodes and services which have the selected tags.
The *View* card and the canvas are updated to display the current selection.

> Users who have the Visualizer Editor permission can save defined views so that any Anypoint Visualizer user can access them.

Reviewer: ```suggestion
For Anypoint Visualizer users to be able to access defined views, Visualizer editors (with Visualizer Editor permission) must have first saved those defined views.
```

Author: Emphasizing permission requirement first since users kept missing this previously.
